### PR TITLE
[Wallet] Fix 8 digit codes for mainnet

### PR DESCRIPTION
**Pull request by i1skn**
### Description

* Fix attestaion service URL, when `/` in the end is missing
* Remove firebase proxies.

### Tested

Verification flow on Android device

### Backwards compatibility

Yes

### CODE REVIEW

**Review by gnardini — APPROVED** [review]
Tested locally on mainnet and is working for me 😄

**Comment by nambrot** [comment]
Would note that Mainnet isn't the best testing ground for this as most validators haven't updated yet. So I'd probably recommend doing it on Alfajores as well

**Comment by i1skn** [comment]
@nambrot this PR specifically fix 8 digit codes on mainnet, on alfajores it works fine without this PR. Also, all validators on mainnet are on >= 1.1.0 version now.

File: packages/sdk/contractkit/src/wrappers/Attestations.ts [hunk 1]
```diff
@@ -331,7 +331,11 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
 
         const nameClaim = metadata.findClaim(ClaimTypes.NAME)
 
-        const resp = await fetch(`${attestationServiceURLClaim.url}status`)
+        const resp = await fetch(
+          `${attestationServiceURLClaim.url}${
+            attestationServiceURLClaim.url.substr(-1) === '/' ? '' : '/'
+          }status`
+        )
```
Comment (jeanregisser) [inline packages/sdk/contractkit/src/wrappers/Attestations.ts:338]: Too bad JavaScript doesn't provide a builtin way to join url parts.

We do have a string helper though which does the same as what you did: https://github.com/celo-org/celo-monorepo/blob/4276a18e2f9bb5dfc9ef96695bae72bc67a9215b/packages/sdk/base/src/string.ts#L1-L7

**Review by jeanregisser — APPROVED** [review]
Thanks! 👍 
Nice seeing one less dependency.

File: packages/sdk/contractkit/src/wrappers/Attestations.ts (hunk 1 above)
Comment (AlexBHarley) [reply packages/sdk/contractkit/src/wrappers/Attestations.ts:338]: Nested deep in the identity package we do https://github.com/celo-org/celo-monorepo/blob/master/packages/sdk/identity/src/offchain/utils.ts#L360.

**Pull request merged**